x/incentive/keeper/scenario_test.go

```diff
@@ -171,6 +179,20 @@ func TestBasicIncentivePrograms(t *testing.T) {
 		rewards,
 		"alice pending rewards at time 300",
 	)
+	// actually claim the rewards (same amount)
+	rewards, err = k.UpdateAccount(k.ctx, alice)
+	require.NoError(k.t, err)
+	require.Equal(
+		k.t,
+		sdk.NewCoins(sdk.NewInt64Coin(umee, 100000+7_920000+8_000000)),
```

Comment:
1. this numbers are repeated, let's save it in a variable
2. could you calculate this numbers, instead of hardcoding the sum of values?

Reply:
Saved to a var. I do want to keep the sum hardcoded, since this test is for a specific scenario and it should fail if we make any unintentional changes